Stack/LargestRectangleAreaHistogram.java

```diff
@@ -0,0 +1,34 @@
+import java.util.*;
+import java.io.*;
+class LargestRectangleAreaHistogram{
+  public static int largestRectangleArea(int[] heights) {
+    int i=0,ans=0,t;
+    Stack<Integer> st=new Stack<>();
+    while(i<heights.length)
+    {
+        if(st.isEmpty()||heights[st.peek()]<=heights[i])
+            st.push(i++);
+        else{
+            t=st.pop();
+            ans=Math.max(ans,heights[t]*(st.isEmpty()?i:i-st.peek()-1));
+        }
+    }
+    while(!st.isEmpty())
+    {
+        t=st.pop();
+            ans=Math.max(ans,heights[t]*(st.isEmpty()?i:i-st.peek()-1));
+    }
+    return ans;
+}
+public static void main (String[] args) {
```

Comment:
The formatting is a bit off. Instead of creating `pvsm` for individual  files can we create one for whole stack folder

Reply:
Should I remove the pvsm then?

Reply:
Yes

Reply:
create some helper method wrap you solution logic in private method

Reply:
Yes, I committed the changes.
Please go through them.

Reply:
Merged